src/initial-submission/components/FileDetailsForm.test.tsx

```diff
@@ -114,12 +112,10 @@ describe('File Details Form', (): void => {
                     initialValues={testInitialValues}
                 />,
             );
-            expect(
-                (container.querySelector('.cover-letter__input') as TextareaHTMLAttributes<HTMLTextAreaElement>).value,
-            ).toBe('');
+            expect(container.querySelector('#coverLetter .ProseMirror').textContent).toBe('');
         });
 
-        it('should call the save mutation with correct variables when cover letter is changed', async (): Promise<
+        it.skip('should call the save mutation with correct variables when cover letter is changed', async (): Promise<
```

Comment:
skip left in

Reply:
It's left as on purpose. It will be addressed in another PR as per discussion with @hdrury1